tools/macos_buildenv.sh

```diff
@@ -15,28 +15,82 @@ realpath() {
     cd "${OLDPWD}" || exit 1
 }
 
+buildenv_from_download_server() { # 2.6
+    http_code=$(curl -sI -w "%{http_code}" "https://downloads.mixxx.org/dependencies/${1}/macOS/${2}.zip" -o /dev/null)
+    if [ "$http_code" -ne 200 ]; then
+        >&2 echo "Downloading  failed with HTTP status code: $http_code"
+        exit 1
+    fi
+    curl "https://downloads.mixxx.org/dependencies/${1}/macOS/${2}.zip" -o "${3}.zip"
+    OBSERVED_SHA256=$(shasum -a 256 "${3}.zip"|cut -f 1 -d' ')
+    if [[ "$OBSERVED_SHA256" == "$4" ]]; then
+        >&2 echo "Download matched expected SHA256 sum $4"
+        echo "${3}.zip"
+    else
+        >&2 echo "ERROR: Download did not match expected SHA256 checksum!"
+        >&2 echo "Expected $4"
+        >&2 echo "Got $OBSERVED_SHA256"
+        exit 1
+    fi
+}
+
+buildenv_from_gh_release() {
+    part=0
+    trap 'find . -name "mixxx-deps-${1}-${3}-${2}.*" -delete && exit' SIGINT SIGABRT
+    part_uri="https://github.com/acolombier/vcpkg/releases/download/${1}-${2}/mixxx-deps-${1}-${3}-${2}.zip.part0${part}"
+    while true; do
+        http_code=$(curl -sI -w "%{http_code}" "$part_uri" -o /dev/null)
+        if [ "$http_code" -ne 200 ] && [ "$http_code" -ne 302 ]; then
+            if [ "$part" = 0 ]; then
+                >&2 echo "Downloading $part_uri failed with HTTP status code: $http_code"
+                exit 1
+            else
+                break
+            fi
+        fi
+        curl -L "${part_uri}" -o "$(basename "${part_uri}")"
+        part=$(( part + 1))
+        part_uri="https://github.com/acolombier/vcpkg/releases/download/${1}-${2}/mixxx-deps-${1}-${3}-${2}.zip.part0${part}"
+    done
+
+    cat mixxx-deps-"${1}-${3}-${2}".zip.part* > "${4}"
+    find . -name "mixxx-deps-${1}-${3}-${2}.zip.part*" -delete
+
+    expected_sha256=$(curl -L "https://github.com/acolombier/vcpkg/releases/download/${1}-${2}/mixxx-deps-${1}-${3}-${2}.zip.sha256")
+    observed_sha256=$(shasum -a 256 "${4}"|cut -f 1 -d' ')
+
+    if [ "$expected_sha256" = "$observed_sha256" ]; then
+        >&2 echo "Download matched expected SHA256 sum"
+        rm -f "mixxx-deps-${1}-${3}-${2}.sha256"
+    else
+        >&2 echo "ERROR: Download did not match expected SHA256 checksum!"
+        find . -name "mixxx-deps-${1}-${3}-${2}.*" -delete
+        exit 1
+    fi
+}
+
 # some hackery is required to be compatible with both bash and zsh
 THIS_SCRIPT_NAME=${BASH_SOURCE[0]}
 [ -z "$THIS_SCRIPT_NAME" ] && THIS_SCRIPT_NAME=$0
 
+VCPKG_COMMIT_SHA="d3b6db3"
+BUILDENV_MINOR_VERSION="2.6"
+
 if [ -n "${BUILDENV_ARM64}" ]; then
     VCPKG_TARGET_TRIPLET="arm64-osx-min1100-release"
-    BUILDENV_BRANCH="2.5-rel"
-    BUILDENV_NAME="mixxx-deps-2.5-arm64-osx-min1100-release-40c29ff"
```

Comment:
In https://github.com/mixxxdj/vcpkg/pull/157 I have proposed to drop min1100 from the file name and add cross in case it is a cross compiled environment. I would prefer to merge my PR first and get around of a second PR that introduces the renaming here. This keep the naming schema on GitHub consistent from the start.